Components/Import/Entity/PlentymarketsImportEntityItemCategoryTree.php

```diff
@@ -189,4 +214,35 @@ public function import()
 			return $parentId;
 		}
 	}
+	
+	/*
+	 * Get the category name from Plenty
+	 */	
+	protected function getLocalCategoryName($categoryId)
+	{
+		$categoryName = false;
+	
+		$Request_GetItemCategoryCatalog = new PlentySoapRequest_GetItemCategoryCatalog();
+		$Request_GetItemCategoryCatalog->CategoryID = $categoryId;
+		$Request_GetItemCategoryCatalog->Lang = $this->plentyLang;
```

Comment:
müsste das nicht `$Request_GetItemCategoryCatalog->Lang = $this->shopLang;` sein?